### DISCUSSION TITLE
Update starlette dependency to 0.36.2 due to vulnerability

### DISCUSSION THREAD

**Discussion by clayk** [fastapi/fastapi#11096] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
https://github.com/encode/starlette/pull/2443
```

### Description

We were alerted by our dependabot alerts that an update for starlette is available due to a Content-Type Header ReDoS.  This is fixed in starlette 0.36.2 and packages <0.36.1 are vulnerable.

Currently when looking at a pipenv graph fastapi is requiring a version <0.36.0.

My question/discussion pertains to updating the requirement to allow the dependency to be updated to a version that will encompass the patched version of starlette.

### Operating System

macOS

### FastAPI Version

0.109.1

### Pydantic Version

2.6.1.

### Python Version

3.9.13

**Comment by n8sty** [comment, marked as answer]
Fixed https://github.com/tiangolo/fastapi/releases/tag/0.109.2
